sklearn/kernel_ridge/kernel_ridge.py

```diff
@@ -81,27 +63,25 @@ class KernelRidge(BaseEstimator, RegressorMixin):
       "Machine Learning: A Probabilistic Perspective", The MIT Press
       chapter 14.4.3, pp. 492-493
 
+    * An, Senjian, Wanquan Liu, and Svetha Venkatesh.
+      "Face recognition using kernel ridge regression." Computer Vision
+      and Pattern Recognition, 2007. CVPR'07. IEEE Conference on. IEEE,
+      2007.
+
     See also
     --------
     sklearn.linear_model.Ridge:
         Linear ridge regression.
     sklearn.svm.SVR:
         Support Vector Regression implemented using libsvm.
+    sklearn.svm.SVC:
```

Comment:
I don't think this is relevant, but you should add KernelRidgeClassifier to KernelRidge's see also

Reply:
Done